draft-ietf-opsawg-pcaplinktype.md

```diff
@@ -748,20 +749,21 @@ This is the initial table for the registry:
 
 {::include linktypes.md}
 
+### Guidance for Designated Experts {#sec-de}
 
-### Guidance for Registration
+When processing a request for an allocation, the Designated Experts will encourage the requester to provide a specification at a stable URL.
 
-When processing a request for an allocation, the requester will be asked to provide a specification at a stable URL.
-There is no requirement for a specification.
+There is no requirement for a specification, but often review of the specification allows the Designated Expert to determine if the allocation actually is a duplicate of another specification.
 
 When the contents of the link type can contain an IPv4 or IPv6 header, then the octets between the beginning of the link type and the IP header needs to be clearly specified.
```

Comment:
I believe there was a DISCUSS comment from Ketan on this statement for better clarification on what it means.

Reply:
I'm not really sure I know what is lacking, but here is a an attempt: #193